app/code/community/Bolt/Boltpay/controllers/Adminhtml/Sales/Order/CreateController.php

```diff
@@ -16,7 +16,6 @@
  */
 
 require_once(Mage::getModuleDir('controllers','Mage_Adminhtml').DS.'Sales'.DS.'Order'.DS.'CreateController.php');
-require_once(Mage::getModuleDir('controllers','Bolt_Boltpay').DS.'OrderControllerTrait.php');
```

Comment:
Does this `require_once` need to be replaced for the renamed trait?

Reply:
@matt-thomason It is not necessary.  autoload with find it with the new name.